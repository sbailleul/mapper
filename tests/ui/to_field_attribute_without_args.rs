use mapper_impl::Mapper;

#[derive(Mapper)]
pub struct User{
    #[to()]
    pub name: String
}

pub struct Person{
    pub name: String
}

fn main(){}