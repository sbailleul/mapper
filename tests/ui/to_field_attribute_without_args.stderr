error: To field attribute destination type should be specified at first position
 --> tests/ui/to_field_attribute_without_destination.rs:5:10
  |
5 |     #[to()]
  |          ^
